### DISCUSSION TITLE
Maintanance mode

### DISCUSSION THREAD

**Discussion by QVinto** [YetiForceCompany/YetiForceCRM#15623] [Wishlist]
Hello,
it would be great to have an option like maintanance mode when no user can login into system and all crons will be stopped.
Login screen will show maintanance announcement. 
It is necessary for example when we need to update system. Instead of deactivating all users we just click on that option and everything will be handled by the system itself. Only sysadmin will be able to access the system.

**Comment by mariuszkrzaczkowski** [comment]
I thought about it, but there is not enough time for it

**Reply by QVinto** [reply]
I get it. It was just an idea. Also Login screen maintenance mode is just nice to have. 

But maybe everything else could be done very easy. I dont know, just another idea.

Create in user status new record (Active, Inactive, PAUSED), Puased will have same function like inactive. Then we can make mass edit to change all Active users to Paused and after maintanace we can changed it back for all paused to Active. Only one problem here>Sys admin should always be active no matter what.
![image](https://user-images.githubusercontent.com/4982986/148941634-71a97083-e4bf-4b65-a1ed-e6511295f086.png)

exactly same thing could be done with cron jobs. Just add button do make all Active jobs to Paused jobs and then back (so there will be 3 option Acitve Inactive and Paused). Paused will have same function like inactive. Paused will be jsut like a mark that this process was active before.
![image](https://user-images.githubusercontent.com/4982986/148941849-cb76c5cc-fe89-4be8-8018-c66552aac7d0.png)
